fetch.bs

```diff
@@ -5504,11 +5496,11 @@ constructor must run these steps:
  <var>origin</var>, set <var>window</var> to <var>request</var>'s
  <a for=request>window</a>.
 
- <li><p>If <var>init</var>'s <code>window</code> member is present and it is non-null, then
- <a>throw</a> a <code>TypeError</code>.
+ <li><p>If <var>init</var>["<code>window</code>"] <a for=map>exists</a> and is non-null, then
```

Comment:
If you do `{{RequestInit/window}}` it links, which is nice. Here and elsewhere.

Reply:
Would that produce the quotes as well, or not?

Reply:
Don't think so, just the code/a